conjure/metadata.json

```diff
@@ -0,0 +1,9 @@
+{
+    "friendly-name": "Apache MapReduce",
```

Comment:
This doesn't match the friendly-name from the Makefile.  Does that matter?  If so, I like what you used in the Makefile:  **Apache Hadoop MapReduce**.

Including **Apache** will help differentiate it from our **Bigtop** mapred offering, and including **Hadoop** implies this includes HDFS, which is not necessarily required for mapred.